workflows/MODEL_TO_DATA.nf

```diff
@@ -17,10 +19,12 @@ assert params.email_with_score in ["yes", "no"], "Invalid value for ``email_with
 params.cpus = "4"
 // Default Memory to dedicate to RUN_DOCKER
 params.memory = "16.GB"
-// Scoring Script
-params.scoring_script = "model_to_data_score.py"
-// Validation Script
-params.validation_script = "validate.py"
+// The container that houses the scoring and validation scripts
+params.challenge_container = "sagebionetworks/synapsepythonclient:v4.0.0"
```

Comment:
Wasn't this just a testing container you created? The default configuration won't work using this contianer, right?